requirements.txt

```diff
@@ -1,3 +1,3 @@
 gitpython
-freebsd_sysctl==0.0.5
+freebsd_sysctl==0.0.6
```

Comment:
there's no official release https://github.com/gronke/py-freebsd_sysctl/releases

Reply:
Release tag is published. Thanks for the heads up!